### PR TITLE
Week4 Session 4 Assignment Solution.

### PR DESCRIPTION
**Pull request by rheeturajmishra23**
Hi @kunal8411 , I have raised this PR against the Session 4 assignement which is related to advanced HTML. Please check and provide review comments if required.

**Discarding this PR as I need to reorganize the file structure and branch name.**

### CODE REVIEW

**Comment by kunal8411** [comment]
Checking......

File: Week-2/hometown webpage.html.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kolkata</title>
+</head>
+<body>
+    <h1>Kolkata, India</h1>
+    <section>
+        <h2>Location</h2>
+        <p>Kolkata also known as Calcutta (which was the official name until 2001) is the capital of the Indian state of West Bengal. 
+            <br>It is on the eastern bank of the Hooghly River 80 km (50 mi) west of the border with Bangladesh.</p>
```
Comment (kunal8411) [inline Week-2/hometown webpage.html.html:14]: Avoid the Br tag, leverage the block level tags of HTML. Its better to use block scope elements than the br tag. Specially I am not big fan of the br tag and in the industry also, its reflecting the lack of knowledge if you use br tag very often.

Please consider this for the future, when you will be writing the production code. Thanks

File: Week-2/hometown webpage.html.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    </section>
+        <h2>About</h2>
+        <section>
+            <h3>Weather</h3>
+            <p>Kolkata is subject to a tropical wet-and-dry climate.<br>
+                According to a United Nations Development Programme report, its wind and cyclone zone is "very high damage risk"</p>
+        </section>
+        <section>
+            <h3>Population</h3>
+            <p>According to the 2011 Indian census, Kolkata is the seventh-most populous city in India, with a population of 45 lakh (4.5 million) residents within the city. <br>
```
Comment (kunal8411) [inline Week-2/hometown webpage.html.html:24]: same br tag

File: Week-4/formForJoiningASocialNetwork.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Social Network</title>
+    <style>
```
Comment (kunal8411) [inline Week-4/formForJoiningASocialNetwork.html:8]: This is ok for now but going forward create seperate CSS file and then import the same in HTML.

**Review by kunal8411 — CHANGES_REQUESTED** [review]
LGTM, Added some small corrections, for better code practice. Its ok for now, but it will help you to write good code in the future. Thanks.

File: Week-2/hometown webpage.html.html (hunk 1 above)
Comment (rheeturajmishra23) [reply Week-2/hometown webpage.html.html:14]: Ok I will keep this in mind. From now onwards.

Comment (rheeturajmishra23) [reply Week-2/hometown webpage.html.html:14]: Ok I will keep this in mind. From now onwards.

Comment (rheeturajmishra23) [reply Week-2/hometown webpage.html.html:14]: Ok I will keep this in mind. From now onwards.

File: Week-4/formForJoiningASocialNetwork.html (hunk 3 above)
Comment (rheeturajmishra23) [reply Week-4/formForJoiningASocialNetwork.html:8]: Sure.

**Pull request closed without merging**